test/lib/kitten_blue/jwk/x509_test.exs

```diff
@@ -0,0 +1,113 @@
+defmodule KittenBlue.Jwk.X509Test do
```

Comment:
This library seems to capitalize JWK/JWS/JWT. 
So `KittenBlue.JWK.X509Test` might be more appropriate here. (Same goes for jws's one)

Reply:
oops, thanks